contents/handbook/future.md

```diff
@@ -12,14 +12,14 @@ What motivates us is building an epic product and company.
 
 We're excited by:
 
-* figuring out how far we can go
```

Comment:
I think some of these are purely stylistic (James' writing style) which adds some character